### PR TITLE
v1.13 Backports 2024-05-24

### PR DESCRIPTION
**Pull request by YutaroHayakawa**
* [x] #32084 (@pchaigno) :warning: resolved conflicts
 * [x] #32328 (@joestringer)
 * [ ] #32464 (@darox) :warning: resolved conflicts
 * [x] #32553 (@michi-covalent) :warning: resolved conflicts
 * [x] #32564 (@jibi) :warning: resolved conflicts
 * [x] #32548 (@squeed)
 * [x] #32570 (@mhofstetter)
 * [x] #32600 (@mhofstetter) :warning: resolved conflicts
 * [ ] #32302 (@marseel) :warning: resolved conflicts
 * [ ] #32629 (@antonipp) :warning: resolved conflicts

PRs skipped due to conflicts:

 * #32307 (@ferozsalam)
 * #32450 (@jschwinger233)

Once this PR is merged, a GitHub action will update the labels of these PRs:
```upstream-prs
 32084 32328 32464 32553 32564 32548 32570 32600 32302 32629
```

### CODE REVIEW

**Review by mhofstetter — APPROVED** [review]

**Review by lmb — APPROVED** [review]

**Review by pchaigno — CHANGES_REQUESTED** [review]
It looks like you backported my PR twice, with the second backport being empty.

File: api/v1/Makefile [hunk 1]
```diff
@@ -3,7 +3,7 @@
 include ../../Makefile.defs
 
 # Update this via images/scripts/update-cilium-builder-image.sh
-CILIUM_BUILDER_IMAGE=quay.io/cilium/cilium-builder:57c70665a73759079bfcec1c64b0601d5168993b@sha256:758b172c5362a0aff7227274a88474bc1cd18f8f49715c6d630df2946f030f1f
+CILIUM_BUILDER_IMAGE=quay.io/cilium/cilium-builder:fcf294a0475e9d8b1c8245b97a7e991c20c6ea40
```
Comment (rolinh) [inline api/v1/Makefile]: This looks wrong :thinking:

**Review by jibi — APPROVED** [review]

**Review by joestringer — APPROVED** [review]

**Review by michi-covalent — APPROVED** [review]

File: api/v1/Makefile (hunk 1 above)
Comment (YutaroHayakawa) [reply api/v1/Makefile]: Yep, should be fixed now.

**Comment by YutaroHayakawa** [comment]
> It looks like you backported my PR twice, with the second backport being empty.

Looks like I messed it up when I redo the process once. Fixed now.

**Comment by YutaroHayakawa** [comment]
/test-backport-1.13

**Comment by YutaroHayakawa** [comment]
Rebasing to pull-in bfbb9be3ed5f21ee8c4a9452c139d22578894aef.

**Review by marseel — APPROVED** [review]

**Review by darox — APPROVED** [review]
my change looks good

**Review by pchaigno — APPROVED** [review]
Thanks!

**Review by squeed — APPROVED** [review]

**Comment by YutaroHayakawa** [comment]
/test-1.17-4.19

**Comment by YutaroHayakawa** [comment]
/test-1.19-4.19

**Comment by YutaroHayakawa** [comment]
/test-1.25-4.19

**Review by rolinh — APPROVED** [review]

**Comment by YutaroHayakawa** [comment]
/test-1.17-4.19

**Pull request merged**